README.md

```diff
@@ -12,34 +12,32 @@ Bozza ver. 01.047 – 2020-04-12 &ndash; Scritto con [***Mark Text***](https://m
 
-- Link all'account, che contiene pubblicazioni in relazione a questo manifesto:
+- Link all'account che contiene pubblicazioni in relazione a questo manifesto:
   
   https://mondiaspora.net/people/14281b90538e0138a38a03f7e1ed1bc8
 
 - Link diretto alla breve presentazione di questo manifesto:
   
   https://mondiaspora.net/posts/ab05452057880138212d0e36a79e7653
   
-  Non abbiamo per ora un nostro sito web con base dati, onde proporre una raccolta di firme, una sorta di petizione.  
-  **Commentare [quella pubblicazione](https://mondiaspora.net/posts/ab05452057880138212d0e36a79e7653) in Diaspora è al momento un modo per segnalarci la vostra approvazoine.** Se e quando vorrete farlo, potrete scrivere qualche cosa come:  
+  Ancora non abbiamo predisposto un nostro sito web con base dati, dove proporre una raccolta di firme o petizione di sorta.  
+  **Commentare [quella pubblicazione](https://mondiaspora.net/posts/ab05452057880138212d0e36a79e7653) in Diaspora è al momento il modo di supportare il nostro lavoro e segnalarci la vostra approvazoine.** Se vorrete farlo, ci potrete scrivere qualche cosa come:  
   **"Approvo il manifesto *Nuovo Paradigma 1.0* (bozza ver. nn.nnn)"**.  
-  La versione che vi è piaciuta potrete anche copiarvela ovviamente (così come qualunque altra).  
-  **Se avete una "facciata" nel web, potrete segnalare nella stessa il manifesto.**  
-  Spendere 5 o 10 minuti del vostro tempo per aprire un acount gratuito su Diaspora sarà l'occasione di uscire un po' dalle grinfie di Facebook, con la sua censura e la sua connivenza con *Big Data*.
+  La versione che più vi è piaciuta potrete anche copiarla (e lo stesso dicasi per qualunque altra).  
+  **Se disponete di account, nick o una qualunque "facciata web" potrete segnalare l'esitenza del manifesto tramite la stessa.**  
+  Vi ringrazieremo se spenderete 5-10 minuti del vostro tempo per aprire un account gratuito su Diaspora e sarà anche l'occasione di uscire un po' dalle grinfie di Facebook, con la sua censura e la sua connivenza con *Big Data*.
 
 &nbsp;
 
-Almeno durante un primo periodo, questo sarà un lavoro in evoluzione, anche se vorremo marcare il raggiungimento di certe tappe di sviluppo del pensiero e dell'azione (potremmo valutare se effettuare delle *release*).
+Almeno per un primo periodo questo manifesto é da cosiderare come un lavoro in evoluzione, anche se vorremmo marcare il raggiungimento di certe tappe di sviluppo del pensiero e dell'azione entro breve tempo (potremmo valutare se effettuare delle *release*).
 
 **SE AVETE DELLE IDEE CON CUI CONTRIBUIRE, FATEVI AVANTI.**
 
-Abbiamo per ora una sola regola: **niente aggressioni**, **niente stroncature di nessun apporto**. Solo così la creatività di tutti potrà dare i frutti migliori.
+Abbiamo, per ora, una sola chiara regola: **niente aggressioni**, **niente stroncature o commenti di nessun apporto**, affinché la creatività di tutti potrà portare ai frutti migliori.
```

Comment:
In questa sinceramente preferisco l'originale, "niente commenti" impedisce la discussione, come faciamo a valutare gli apporti? "potrà portare" molte 'p', potrà dare più fluido, e i frutti sono accusativo, oggetto, non dativo, complemento figurato di moto a luogo.

A volte io stesso nel flusso cambio cose e poi mi rendo conto che preferivo la versione precedente, succede normalmente quando uno sta scrivendo... ti invito a rivalutarlo.

Reply:
Avevo aggiunto commenti per rinforzare quel "nessun apporto" che a me suonava poco solido, forse persino troppo aperto. Per me l'originale puó andare bene, solo chiarirei meglio cosa vuoi dire con "nessun apporto"

Reply:
La frase è "**niente stroncature** di nessun apporto", cioè qualunque sia il contenuto dell'apporto, del suggerimento che ci viene dato, non sarà stroncato, non ci sarà aggressione o commento negativo del tipo "ma cosa dice 'sto scemo". Capito? È una regola d'oro per il *brainstorming*.